schema/v0_1_0/mzqc_0_1_0.schema.json

```diff
@@ -215,14 +210,15 @@
 			"description": "Element describing a controlled vocabulary.",
 			"type": "object",
 			"properties": {
+				"namespace": {
+					"description": "PURL namespace of the controlled vocabulary.",
```

Comment:
Good point, I'm not familiar with the precise nomenclature, but to avoid confusion we should use the proper terms.

I'm not sure how to sync offline controlled vocabularies and PURLS. The obvious approach would require custom parsing, which I really want to avoid. It would be stupid if this schema change makes other tasks much more cumbersome.